### ISSUE TITLE
3.11.0RC1: undefined symbol: upb_strtable_lookupptr

### ISSUE THREAD

**Issue by remicollet** [protocolbuffers/protobuf#6918]
Build:
```
/dev/shm/BUILD/php-pecl-protobuf-3.11.0~RC1/NTS/protobuf.c: In function 'get_proto_enumdesc':
/dev/shm/BUILD/php-pecl-protobuf-3.11.0~RC1/NTS/protobuf.c:227:8: warning: implicit declaration of function 'upb_strtable_lookupptr'; did you mean 'upb_inttable_lookupptr'? [-Wimplicit-function-declaration]
   if (!upb_strtable_lookupptr(&proto_to_enumdesc_map_persistent,
        ^~~~~~~~~~~~~~~~~~~~~~
        upb_inttable_lookupptr

```

Run:
`PHP Warning:  PHP Startup: Unable to load dynamic library '/dev/shm/BUILDROOT/php-pecl-protobuf-3.11.0~RC1-1.fc29.remi.7.2.x86_64/usr/lib64/php/modules/protobuf.so' (tried: /dev/shm/BUILDROOT/php-pecl-protobuf-3.11.0~RC1-1.fc29.remi.7.2.x86_64/usr/lib64/php/modules/protobuf.so (/dev/shm/BUILDROOT/php-pecl-protobuf-3.11.0~RC1-1.fc29.remi.7.2.x86_64/usr/lib64/php/modules/protobuf.so: undefined symbol: upb_strtable_lookupptr), /usr/lib64/php/modules//dev/shm/BUILDROOT/php-pecl-protobuf-3.11.0~RC1-1.fc29.remi.7.2.x86_64/usr/lib64/php/modules/protobuf.so.so (/usr/lib64/php/modules//dev/shm/BUILDROOT/php-pecl-protobuf-3.11.0~RC1-1.fc29.remi.7.2.x86_64/usr/lib64/php/modules/protobuf.so.so: cannot open shared object file: No such file or directory)) in Unknown on line 0
`

**Comment by rafi-kamal** [comment]
Thanks for letting us know! This should be fixed in 3.11.0-RC2 release.

**Comment by remicollet** [comment]
Indeed RC2 is fixed

BTW, still not compatible with upcoming 7.4.0 (next week)
Not even really compatible with 7.x  (see pr #6226 ...)
